docs/geedocs/docsrc/answer/geeServerAdmin/createPortableGlobesMaps.rst

```diff
@@ -218,40 +213,44 @@ Create Portable Globes and Maps
          browser. This method works for cutting 2D maps from your
          browser.
 
-      #. By default, you draw a polygon by hand so the **Select Region**
-         drop-down list is set to **Manual**.
       #. Use the |Pan tool| **Hand** tool to pan, then, using the navigation
          controls in the plug-in, zoom in to the region of interest.
       #. To use your mouse to define the polygon, click the |Polygon icon| polygon icon
          in the globe window.
       #. Click on the map or globe to define each point. You can use the
          navigation controls on the right to move the globe or change
          zoom levels while drawing.
-      #. Click the final point at the point of origin to complete the polygon selection.
+      #. Double-click the final point to complete the polygon selection.
+         The polygon turns orange.
 
          |Polygon example|
 
-      #. If you need to redraw the polygon, click **Clear** to delete the polygon you just created.
+      #. To modify the polygon, click and drag the white edit points.
+      #. To delete a vertex, click and hold the mouse on the point until
+         it is removed from the map.
+      #. To redraw the polygon, press the **Esc** key and begin again.
+      #. To save the polygon, press the **Enter** key.
 
       .. rubric:: Defining the polygon with KML
 
       You can also use KML to define the polygon(s). The KML should be

```

Comment:
This is technically not required. If you want, you can simply enter a single polygon element without the complete KML document. I think that's probably worth pointing out since the polygon by itself is a lot less text for the user to keep up with.